week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +47,14 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    arr[i] = Number(arr[i].toFixed(2));
+    if (arr[i] >= 100) arr[i] = 100;
+    arr[i] = arr[i] + "%";
+  }
+  return arr;
+}
 
```

Comment:
Here, the code your write here with new methods and use if else statements, to make your code more clear to the reader

So the following functions below:
```js
function formatPercentage(arr) {
  for (let i = 0; i < arr.length; i++) {
    arr[i] = Number(arr[i].toFixed(2));
    if (arr[i] >= 100) arr[i] = 100;
    arr[i] = arr[i] + "%";
  }
  return arr;
}
```

Can be written as:
```js
function formatPercentage(arr) {
  const newArr = []

  for (let i = 0; i < arr.length; i++) {
    const number = arr[i]

    if (number > 100) {
      newArr.push("100%")
    } else if (Number.isInteger(number)) {
      newArr.push(`${number}%`)
    } else {
      newArr.push(`${number.toFixed(2)}%`)
    }
  }

  return newArr;
}
```